port/common/omrport.tdf

```diff
@@ -1075,3 +1075,6 @@ TraceExit=Trc_PRT_signal_omrsig_is_master_signal_handler_exiting Group=signal Ov
 
 TraceException=Trc_PRT_readCgroupFile_fgets_failed Group=sysinfo Overhead=1 Level=3 NoEnv Template="readCgroupFile: fgets failed for %s with errno=%d"
 TraceException=Trc_PRT_isRunningInContainer_fgets_failed Group=sysinfo Overhead=1 Level=3 NoEnv Template="isRunningInContainer: unexpected format of file %s with errno=%d"
+
+TraceEvent=Trc_PRT_signal_mapPortLibSignalToOSSignal_ERROR_unknown_signal Group=signal Overhead=1 Level=3 NoEnv Template="omrsignal: mapPortLibSignalToOSSignal: ERROR, unknown signal, portLibSignal=0x%X" 
```

Comment:
Should be a TraceException. As should Trc_PRT_signal_mapPortLibSignalToUnix_ERROR_unknown_signal.

Reply:
updated definition of `Trc_PRT_signal_mapPortLibSignalToOSSignal_ERROR_unknown_signal`. will set `Trc_PRT_signal_mapPortLibSignalToUnix_ERROR_unknown_signal` to obsolete and instead, use `Trc_PRT_signal_mapPortLibSignalToUnix_ERROR_unknown_signal`.

Reply:
i will add a commit to replace `Trc_PRT_signal_mapPortLibSignalToUnix_ERROR_unknown_signal` with `Trc_PRT_signal_mapPortLibSignalToOSSignal_ERROR_unknown_signal` in the next set of changes (**Part 2**).